crates/rustc_codegen_spirv/src/linker/new_structurizer.rs

```diff
@@ -168,11 +191,7 @@ impl Structurizer<'_> {
             let block_id = self.func.blocks()[block].label_id().unwrap();
             let terminator = self.func.blocks()[block].instructions.last().unwrap();
             let mut region = match terminator.class.opcode {
-                Op::Return | Op::ReturnValue | Op::Kill | Op::Unreachable => Region {
-                    merge: block,
-                    merge_id: block_id,
-                    exits: indexmap! {},
-                },
+                Op::Return | Op::ReturnValue | Op::Kill | Op::Unreachable => Region::Divergent,
```

Comment:
Just a note here that this will conflict with https://github.com/EmbarkStudios/rust-gpu/pull/563, and it might not be immediately obvious if a merge goes wonky here, so note to double check a merge goes well